diff --git a/MoireDet/lib/models/modules/resnet.py b/MoireDet/lib/models/modules/resnet.py
index 8d6948b..d4e39f3 100644
--- a/MoireDet/lib/models/modules/resnet.py
+++ b/MoireDet/lib/models/modules/resnet.py
@@ -2,6 +2,9 @@
 # @Time    : 2019/8/23 21:55
 # @Author  : zhoujun
 import torch.nn as nn
-from torchvision.models.utils import load_state_dict_from_url
-
+try:
+    from torchvision.models.utils import load_state_dict_from_url
+except ModuleNotFoundError:
+    from torch.hub import load_state_dict_from_url
+
 __all__ = ['ResNet', 'resnet6', 'resnet10', 'resnet18', 'resnet34', 'resnet50', 'resnet101',
diff --git a/MoireDet/lib/models/modules/resnet_dct.py b/MoireDet/lib/models/modules/resnet_dct.py
index 0aae224..9c3e652 100644
--- a/MoireDet/lib/models/modules/resnet_dct.py
+++ b/MoireDet/lib/models/modules/resnet_dct.py
@@ -2,6 +2,9 @@
 # @Time    : 2019/8/23 21:55
 # @Author  : zhoujun
 import torch.nn as nn
-from torchvision.models.utils import load_state_dict_from_url
-
+try:
+    from torchvision.models.utils import load_state_dict_from_url
+except ModuleNotFoundError:
+    from torch.hub import load_state_dict_from_url
+
 __all__ = ['ResNet', 'resnet18_dct', 'resnet34_dct', 'resnet50', 'resnet101',
